remix.init/README.md

```diff
@@ -37,15 +37,29 @@ Ensure all packages are installed by running:
 npm install
 ```
 
-The Netlify CLI starts your app in development mode, rebuilding assets on file changes.
+### Using the Remix Dev Server
```

Comment:
@stephmarie17, I've improved the DX for the Netlify Functions template and that brings along changes to the README. Do you think this could be improved?